.gitignore

```diff
@@ -20,3 +20,6 @@ crates/oxc_napi/*.node
 /*.jsx
 /*.ts
 /*.tsx
+
+# MacOS
+.DS_Store
```

Comment:
nit: this belongs in your global `.gitignore_global`

Reply:
:+1: